tasks/main.yml

```diff
@@ -15,6 +15,7 @@
   delegate_to: localhost
   get_url:
     url: "https://github.com/aelsabbahy/goss/releases/download/{{ goss_version }}/goss-linux-amd64"
+    sha256sum: "{{ goss_sha256sum }}"
```

Comment:
I like the idea of checking the sha256sum, but the implementation should use the list in vars/main.yml with the goss_version provided in the group_vars of the user with your new version from defaults/main.yml if the user does not care. To implement this, this line should read:

```yaml
sha256sum: "{{ goss[goss_version].sha256sum }}"
```